docs/release-notes.md

```diff
@@ -2192,7 +2192,7 @@ https://docs.appcircle.io/build/building-android-applications/android-signing-fo
 
 ### :muscle:Improvement
 
-- Repository Connection now has a new look and feel! Refer to [this documentation](https://docs.appcircle.io/build/adding-a-build-profile#connect-your-repository) to see the new connection screens or dive right into the connection module to check our new simplified experience!
+- Repository Connection now has a new look and feel! Refer to [this documentation](https://docs.appcircle.io/build/connection-guides#connect-your-repository) to see the new connection screens or dive right into the connection module to check our new simplified experience!
```

Comment:
We should also check "anchors" for the changed titles in the PR.

For instance, 👇 
<img width="652" height="369" alt="Screenshot 2025-09-16 at 09 09 35" src="https://github.com/user-attachments/assets/eb40bded-98ce-423a-9837-aa4487d64437" />

@atkncamurcu Please carefully review this PR.